src/features/calendar/components/EventCluster/index.stories.tsx

```diff
@@ -9,9 +9,9 @@ import Single from './Single';
 export default {
   component: Single,
   title: 'Calendar',
-} as ComponentMeta<typeof Single>;
+} as Meta<typeof Single>;
 
-const SingleEvent: ComponentStory<typeof Single> = (args) => (
+const SingleEvent: StoryFn<typeof Single> = (args) => (
```

Comment:
Ah it was ok to update the existing stories like this! Nice!

Reply:
Yeah, that's how I interpret https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#decoratorfn-story-componentstory-componentstoryobj-componentstoryfn-and-componentmeta-typescript-types and it seems to work.